docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,110 @@
+OSG Hosted CE
+=============
+
+
+## Overview
+
+As an alternative to setting up your own CE, OSG offers a hosted CE option
+to connect your existing campus HPC/HTC cluster to the Open Science Grid.
+The OSG team will host and operate the HTCondor compute element which routes user jobs to your cluster,
+configured for the science communities that you choose to support.
+
+![managed services diagram](/compute-element/img/screenshot_2983.png)
+
+
+## Requirements
+
+Here are the requirements:
```

Comment:
```suggestion
Before preparing your cluster for OSG jobs, consider the following points:
```

Reply:
"consider the following points" does not seem strong enough. This is stuff you gotta do.

Reply:
That's a fair point, I just wanted to expand on the context more and copy/pasted what we had in the CE installation guide. How about this?

> Before preparing your cluster for OSG jobs, consider the following requirements:

Reply:
Sounds fine. Can you update the suggestion?